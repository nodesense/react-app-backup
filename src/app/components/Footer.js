/*
  Learn about properties, default props
*/

import React from "react";
import PropTypes from "prop-types";

//import "./footer.css";

export default function Footer(props) { 
        return (
            <div className="highlight"> 
            <hr />
             <p>Copyrights@{props.year}, {props.company}</p>
            </div>
        )
} 


Footer.defaultProps = {
    
}

Footer.propTypes = {
    
}