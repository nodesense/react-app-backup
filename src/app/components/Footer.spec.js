import React from 'react';
import Footer from './Footer';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    // a virtual dom in json
    const tree = renderer
      .create(<Footer year={2019} company="NodeSense" ></Footer>)
      .toJSON();
    expect(tree).toMatchSnapshot();
    expect(tree).toMatchSnapshot();
  });