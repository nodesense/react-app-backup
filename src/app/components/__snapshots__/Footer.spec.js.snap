// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="highlight"
>
  <hr />
  <p>
    Copyrights@
    2019
    , 
    NodeSense
  </p>
</div>
`;

exports[`renders correctly 2`] = `
<div
  className="highlight"
>
  <hr />
  <p>
    Copyrights@
    2019
    , 
    NodeSense
  </p>
</div>
`;
